test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -129,6 +129,21 @@ describe('33acrossBidAdapter:', function () {
       return this;
     };
 
+    this.withProductAndGPID = (prod = 'siab', gpid) => {
```

Comment:
since we already have a `withProduct` function, I think it's better to just have a `withGpid` function... I know that Object.assign could be problematic because it's a shallow copy but you could probably use `util.mergeDeep`
e.g.
```
this.withProduct = (prod = 'siab') => {
      ttxRequest.imp.forEach((imp) => {
        utils.mergeDeep(imp, {
          ext: {
            ttx: {
              prod
            }
          }
        });
      });

      return this;
    };
    
this.withGpid= (gpid) => {
      ttxRequest.imp.forEach((imp) => {
        utils.mergeDeep(imp, {
          ext: {
            gpid
          }
        });
      });

      return this;
    };    
```